Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1995,7 +1995,7 @@ estimatesFlow estimatedQuotes state = do
         Nothing -> 0
       showRateCardIcon = if (null estimateFareBreakup) then false else true
       zoneType = getSpecialTag $ if isJust (estimatedVarient !! 0) then (fromMaybe dummyEstimateEntity (estimatedVarient !! 0)) ^. _specialLocationTag else Nothing
-  if ((not (null estimatedVarient)) && (isLocalStageOn FindingEstimate)) then do
+  if not (null estimatedVarient) then do
```

Comment:
why this Local stage check is removed?

Reply:
Resolved, there is no need of removing this check